### PR TITLE
add pybboxes package

### PR DESCRIPTION
**Pull request by fcakyon**
@conda-forge/help-python|

ready for review!

### CODE REVIEW

**Comment by fcakyon** [comment]
@conda-forge/help-python|

ready for review!

**Comment by fcakyon** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by fcakyon** [comment]
@isuruf do you know why checks fail?

**Comment by fcakyon** [comment]
Hey @weiji14, sorry for bothering you. Would you help me passing tests for this package? :sweat_smile: I am pinging you since you were the one helping me for SAH conda-forge package :rocket: I

**Review by weiji14 — COMMENTED** [review]
> Hey @weiji14, sorry for bothering you. Would you help me passing tests for this package? sweat_smile I am pinging you since you were the one helping me for SAHI conda-forge package rocket

Sure, I can spare a few minutes. Maybe try adding `pyhumps` as a host dependency as suggested below. If you want to test things locally, follow instructions at https://conda-forge.org/docs/maintainer/updating_pkgs.html#testing-changes-locally.

**Review by carterbox — CHANGES_REQUESTED** [review]
@fcakyon, please do not remove our [pull-request checklist ](https://github.com/conda-forge/staged-recipes/blob/main/.github/pull_request_template.md)from the top of your PR. I have restored it for you. All of these things must be complete before your PR is ready to be merged. For example, not all of the persons listed in maintainers has commented.

Your recipe has also not successfully built yet. Do ask specific questions if you need help in the future instead of just "ready for review".

This recipe is waiting on https://github.com/conda-forge/pyhumps-feedstock/issues/2 to be resolved.

**Comment by fcakyon** [comment]
@devrimcavusoglu, are you willing to be added as a maintainer for the pybboxes conda-forge distribution?

File: recipes/pybboxes/meta.yaml
Comment (fcakyon) [reply recipes/pybboxes/meta.yaml:19]: @weiji14 unfortunately it still gives pyhumps error

**Comment by fcakyon** [comment]
@carterbox I have no idea why tests are failing? Is there anyone that can tell me what is missing?

**Comment by fcakyon** [comment]
Can someone guide me on what to change to pass the tests? @conda-forge/staged-recipes I have read all the knowledge base documentation.

**Comment by carterbox** [comment]
> This recipe is waiting on https://github.com/conda-forge/pyhumps-feedstock/issues/2 to be resolved.

**Comment by carterbox** [comment]
Please ping the team again when this issue is resolved.

File: recipes/pybboxes/meta.yaml
Comment (devrimcavusoglu) [inline recipes/pybboxes/meta.yaml:2]:
```suggestion
{% set version = "0.1.3" %}
```

File: recipes/pybboxes/meta.yaml
Comment (devrimcavusoglu) [inline recipes/pybboxes/meta.yaml:20]:
```suggestion
```

File: recipes/pybboxes/meta.yaml
Comment (devrimcavusoglu) [inline recipes/pybboxes/meta.yaml:24]:
```suggestion
```

File: recipes/pybboxes/meta.yaml
Comment (devrimcavusoglu) [inline recipes/pybboxes/meta.yaml:10]:
```suggestion
  sha256: 18d7cddb7c276aada2f498c916099c7c623a712d6380bcbdcb1ebb803ffda791
```

**Review by devrimcavusoglu — CHANGES_REQUESTED** [review]
Fields need to be updated

**Comment by fcakyon** [comment]
@conda-forge/help-python ready fro review!

**Pull request closed without merging**
